zuul.d/project.yaml

```diff
@@ -122,9 +122,7 @@
               - playbooks/ansible-test-network-integration-base/files/bootstrap-splunk.yaml
               - playbooks/ansible-network-appliance-base/.*
               - roles/splunk.*
-        - ansible-security-trendmicro-deepsec-appliance:
-            files:
-              - playbooks/ansible-network-appliance-base/.*
+        - ansible-test-security-integration-trendmicro-deepsec-python38
```

Comment:
You can also comment out the vmware and openvswitch jobs. Just to save some resources.

Reply:
You did the change the `gate` section, you should actually add this line in the `check` part, e.g around line 85..